extension/src/cli/dvc/constants.ts

```diff
@@ -28,9 +28,11 @@ export enum SubCommand {
 }
```

Comment:
Looks like the row borders are now a bit buggy which I believe is due to [this line in our styles](https://github.com/iterative/vscode-dvc/blob/84a9883203a063265cad7aab8d56220634d8498c/webview/src/experiments/components/table/styles.module.scss#L239):

<img width="638" alt="Screen Shot 2022-09-15 at 7 43 29 AM" src="https://user-images.githubusercontent.com/43496356/190407280-fe2be3df-cfd4-4b18-873a-9aece631aef6.png">

Reply:
If I remember correctly, the CSS lines were added to make the last row on the table not have a border 🤔